testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -256,17 +256,23 @@
   $server.run("rm #{cobbler_log_file}", check_errors: false)
   _result, code = $server.run("test -f #{cobbler_conf_file}.old", check_errors: false)
   if !code.zero?
-    step %(I install package "python3-python-json-logger" on this "server")
     handler_name = 'FileLogger02'
     formatter_name = 'JSONlogfile'
     handler_class = "\"\n[handler_#{handler_name}]\n" \
                   "class=FileHandler\n" \
                   "level=DEBUG\n" \
                   "formatter=#{formatter_name}\n" \
                   "args=('#{cobbler_log_file}', 'a')\n\n" \
-                  "[formatter_#{formatter_name}]\n" \
-                  "format =[%(threadName)s] %(asctime)s - %(levelname)s | %(message)s\n" \
-                  "class = pythonjsonlogger.jsonlogger.JsonFormatter\n\""
+                  "[formatter_#{formatter_name}]\n"
+    if $product == 'Uyuni'
+      step %(I install package "python3-python-json-logger" on this "server")
+      handler_class += "format =[%(threadName)s] %(asctime)s - %(levelname)s | %(message)s\n" \
+                      "class = pythonjsonlogger.jsonlogger.JsonFormatter\n\""
+    else
+      handler_class += "format ={\\''threadName\\'': \\''%(threadName)s\\'', " \
+                      "\\''asctime\\'': \\''%(asctime)s\\'', \\''levelname\\'':  \\''%(levelname)s\\'', " \
+                      "\\''message\\'': \\''%(message)s\\''}\n\""
+    end
```

Comment:
What a heck :D 
Why we have a different format depending the product version?
Are we really talking about Head vs Uyuni?
Or we are talking about new version in uyuni repo compared to old version in spacewalk repository?

Reply:
On SUMA we don't have direct access to a certain package which Uyuni has
Instead of adding yet another repository to the CI, a different formatting for SUMA was  implemented in https://github.com/SUSE/spacewalk/pull/20974